dev/TeachingTip/InteractionTests/TeachingTipTests.cs

```diff
@@ -178,6 +178,37 @@ public void TipCanFollowTarget()
             }
         }
 
+        [TestMethod]
+        public void TipFollowsTargetOnWindowResize()
+        {
+            using (var setup = new TestSetupHelper("TeachingTip Tests"))
+            {
+                elements = new TeachingTipTestPageElements();
+                foreach (TipLocationOptions location in Enum.GetValues(typeof(TipLocationOptions)))
+                {
+                    SetTeachingTipLocation(location);
+
+                    ScrollTargetIntoView();
+                    Wait.ForIdle();
+                    OpenTeachingTip();
+                    double initialTipVerticalOffset = GetTipVerticalOffset();
+                    double initialScrollViewerVerticalOffset = GetScrollViewerVerticalOffset();
+
+                    ScrollBy(10);
+                    WaitForOffsetUpdated(initialScrollViewerVerticalOffset + 10);
+                    Equals(GetTipVerticalOffset(), initialTipVerticalOffset);
+
+                    //Unmaximize then maximize the window, to force a window size changed event.
+                    KeyboardHelper.PressKey(Key.Down, ModifierKey.Windows);
```

Comment:
How does unmaximizing then maximizing help test this?  Shouldn't the test be done before the re-maximize?

Reply:
We scrolled the tip's target by ten pixels, so the tip is no longer positioned correctly. After forcing a WindowSizeChanged event we can test to see if the tip's position was updated as a result. The easiest way to do this is to return the test app to its original state of maximized.

Reply:
I'm not following - but I don't know much about this test infra.  Does the app start out maximized?  If so, won't the tip change position on unmaximize, then change back to the original position on re-maximize, meaning that you can't test the repositioning that late?

Reply:
It starts maximized, but the tip does not usually change positions, so when we call ScrollBy(10) the tip will be miss positioned until the next event that causes the tip to be re-positioned. In this test that event is the window size changed, so when we return to maximum the tip will not longer be miss positioned.